src/schema/annotation.yaml

```diff
@@ -110,6 +112,7 @@ classes:
       - KEGG.ORTHOLOGY prefix is used for KO numbers
     todos:
       - is OrthologyGroup instantiated in an MongoDB collection? Aren't Pathways searchable in the Data Portal?
+    deprecated: "not used. 2024-07-10 https://github.com/microbiomedata/nmdc-schema/issues/1881"
```

Comment:
We do use the KEGG prefix the both of the aggregation tables which is I think an argument for keeping OrthologyGroup which would also mean keeping FunctionalAnnotationTerm. 
i think what should happen is the range of [gene_function_id](https://microbiomedata.github.io/berkeley-schema-fy24/gene_function_id/) should be OrthologyGroup plus a structured_pattern.syntax constraint @turbomam

Reply:
@SamuelPurvine this is what we were talking about on Wednesday, yes?

Reply:
Yep, and is why I'd brought it up :)

Reply:
@aclum @SamuelPurvine @mslarae13 

Making sure I understand various conversations - if you will have a use for OrthologyGroup and parent classes, I should close this PR without merging, leave the classes in the schema, and they will be used after berkeley rollout when you make changes to `gene_function_id`?